### PR TITLE
CircleCI: switched from Starexec-CentOS to Fedora

### PR DESCRIPTION
**Pull request by Tomaqa**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -95,42 +95,44 @@ jobs:
...
-                PARALLEL: ON
+                PARALLEL: OFF
         steps:
             - checkout
             - run:
-                name: Release build gcc under devtoolset-8
-                command: |
-                    cat ./ci/build_maximally_static.sh |scl enable devtoolset-8 bash
+                name: Release build gcc in Fedora
+                command: ./ci/run_travis_commands.sh
```
Comment (blishko) [inline .circleci/config.yml:135]: Can we keep using `build_maximally_static.sh` script? It's useful to know we can build fully static executable.

Comment (Tomaqa) [reply .circleci/config.yml:135]: Then it should be necessary to install also `static` packages in a docker image. What about making it a separate job that would just build and not run tests?

**Comment by Tomaqa** [comment]
I guess that the correct CI jobs will appear after the merge into master ..

File: .circleci/config.yml (hunk 1 above)
Comment (blishko) [reply .circleci/config.yml:135]: I still don't know what you mean by `static` packages.
It was a job that was just building and not running tests, no?

Comment (Tomaqa) [reply .circleci/config.yml:135]: I formerly removed static packages from the Fedora image which were there to allow `build_maximally_static`. I included them again in the image in order to allow this test of static build - as it was before.

Comment (Tomaqa) [reply .circleci/config.yml:135]: https://github.com/usi-verification-and-security/verify-env/pull/4/files

**Comment by Tomaqa** [comment]
I forgot to add the new `build-fedora-release-static` job into workflows - hence the last force-push.

**Review by blishko — APPROVED** [review]
LGTM!

**Pull request merged**